cmd/kubeadm/app/cmd/certs_test.go

```diff
@@ -378,18 +378,21 @@ func TestGenCSRConfig(t *testing.T) {
 
 	// A minimal kubeadm config with just enough values to avoid triggering
 	// auto-detection of config values at runtime.
-	var kubeadmConfig = `
-apiVersion: kubeadm.k8s.io/v1beta3
+	var kubeadmConfig = fmt.Sprintf(`
+apiVersion: %s
 kind: InitConfiguration
 localAPIEndpoint:
   advertiseAddress: 192.0.2.1
 nodeRegistration:
   criSocket: /path/to/dockershim.sock
 ---
-apiVersion: kubeadm.k8s.io/v1beta3
+apiVersion: %s
```

Comment:
nit:
```suggestion
apiVersion: %[1]s
```
So we avoid to repeat the argument twice